server/ot_rpc.go

```diff
@@ -170,6 +171,7 @@ func runTranscode(n *core.LivepeerNode, orchAddr string, httpc *http.Client, not
 	req.Header.Set("Credentials", n.OrchSecret)
 	req.Header.Set("Content-Type", contentType)
 	req.Header.Set("TaskId", strconv.FormatInt(notify.TaskId, 10))
+	req.Header.Set("Pixels", strconv.FormatInt(tData.Pixels, 10))
```

Comment:
I was on the fence about just calling this header name "Pixels" vs. "DecodedPixels". I stuck with "Pixels" for now, but am open to changing it if people think the latter is clearer.